build.gradle

```diff
@@ -123,7 +123,7 @@ check.dependsOn installDist
 
 dependencies {
     compile files("${System.properties['java.home']}/../lib/tools.jar")
-    compile 'org.broadinstitute:gatk:4.alpha.1-129-gef0152f-20160620.203528-1'
+    compile 'org.broadinstitute:gatk:4.alpha.1-146-gfacef8d-SNAPSHOT'
```

Comment:
done

On Thu, Jun 23, 2016 at 2:57 PM, droazen <notifications@github.com> wrote:

> In build.gradle
> <https://github.com/broadinstitute/gatk-protected/pull/582#discussion_r68294514>
> :
>
> > @@ -123,7 +123,7 @@ check.dependsOn installDist
> >
> >  dependencies {
> >      compile files("${System.properties['java.home']}/../lib/tools.jar")
> > -    compile 'org.broadinstitute:gatk:4.alpha.1-129-gef0152f-20160620.203528-1'
> > +    compile 'org.broadinstitute:gatk:4.alpha.1-146-gfacef8d-SNAPSHOT'
>
> Remember to include the full timestamp of the desired snapshot here to
> make the dependency less brittle (otherwise the newest snapshot with this
> version will be used).
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/broadinstitute/gatk-protected/pull/582/files/9e44af717ee69882062f84eb18c1a24c8d37bfeb#r68294514>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AIDEatPvP9IBn3eisIOoK_WdSnU1wgfPks5qOtc1gaJpZM4I7ZOy>
> .
>